.github/workflows/avm.res.network.virtual-hub.yml

```diff
@@ -0,0 +1,83 @@
+name: "avm.res.network.virtual-hub"
```

Comment:
Seems to be an unrelated workflow from a different PR. I'd recommend to remove it from this branch